pkgs/development/haskell-modules/configuration-ghc-8.8.x.nix

```diff
@@ -85,10 +84,7 @@ self: super: {
     url = "https://raw.githubusercontent.com/hvr/head.hackage/master/patches/regex-posix-0.95.2.patch";
     sha256 = "006yli58jpqp786zm1xlncjsilc38iv3a09r4pv94l587sdzasd2";
   });
-  haskell-src-exts = appendPatch super.haskell-src-exts (pkgs.fetchpatch {
-    url = "https://raw.githubusercontent.com/hvr/head.hackage/master/patches/haskell-src-exts-1.21.0.patch";
-    sha256 = "0alb28hcsp774c9s73dgrajcb44vgv1xqfg2n5a9y2bpyngqscs3";
-  });
+  haskell-src-exts = super.haskell-src-exts;
```

Comment:
I think this won't do anything in an overlay and can be removed

Reply:
Thanks, wasn't sure 😃